tpot/tpot.py

```diff
@@ -348,8 +349,9 @@ def export(self, output_file_name):
 '''
         if '_variance_threshold' in operators_used: pipeline_text += 'from sklearn.feature_selection import VarianceThreshold\n'
         if '_select_kbest' in operators_used: pipeline_text += 'from sklearn.feature_selection import SelectKBest\n'
+        if '_select_fwe' in operators_used: pipeline_text += 'from sklearn.feature_selection import SelectFwe\n' # added selectfwe
```

Comment:
No need to have a comment saying `# added selectfwe` -- git and GitHub track those changes for us. :-)